README.md

```diff
@@ -1,3 +1,36 @@
-This is a nascent project for implementing a rich text layout engine backed
-by the [swash](https://github.com/dfrg/swash) crate and implementing the 
-[piet](https://github.com/linebender/piet) text API.
+<div align="center">
+
+# Parley
+
+**Rich text layout**
+
+[![Latest published parley version.](https://img.shields.io/crates/v/parley.svg)](https://crates.io/crates/parley)
```

Comment:
Since we're not yet published, I'd probably comment this out. Although maybe it's fine like this?

Reply:
I uncommented with the assumption that the release is no more than a week away.